xml/vk.xml

```diff
@@ -22172,6 +22172,13 @@ typedef void* <name>MTLSharedEvent_id</name>;
                 <enum value="&quot;VK_KHR_extension_529&quot;"          name="VK_KHR_EXTENSION_529_EXTENSION_NAME"/>
             </require>
         </extension>
+        <extension name="VK_KHR_extension_530" number="530" author="QNX" contact="Mike Gorchak @mgorchak-blackberry" supported="disabled">
```

Comment:
Actually, my bad - should change all the "KHR" in the PR to "QNX" and add the "_QNX" suffix to the flag bit reservation. But structurally it is good.

Reply:
Will do. Cancelling this request now.

Reply:
You realize you can just update it. Creating new tickers is super spammy...